### PR TITLE
Adding Java SDK Tilequery API page

### LINKED ISSUE #606: Add docs on Java SDK's support for Tilequery API
We'll eventually need docs on our Java SDK's support for Tilequery API. Tilequery work moving in https://github.com/mapbox/mapbox-java/pull/860

cc @zugaldia @osana

### PR DESCRIPTION
**Pull request by langsmith**
Resolves #606 by adding a page about the Java SDK's tools for interacting with the Mapbox Tilequery API

screencapture-localhost-8080-android-docs-java-overview-tilequery-2018-11-30-14_56_59

### CODE REVIEW

**Comment by langsmith** [comment]
cc @brsbl as fyi

**Comment by langsmith** [comment]
related demo app pr https://github.com/mapbox/mapbox-android-demo/pull/864

File: src/pages/java/overview/tilequery.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+prependJs:
+  - "import CodeLanguageToggle from '../../../components/code-language-toggle';"
+  - "import ToggleableCodeBlock from '../../../components/toggleable-code-block';"
+  - "import { Floater } from '../../../components/floater';"
+
+---
+
+The Mapbox Tilequery API allows you to retrieve data about specific features from a vector tileset, based on a given latitude and longitude. The API makes it possible to query for features within a radius, do point in polygon queries, query for features in multiple composite layers, and augment data from the Mapbox Geocoding API with custom data.
+
+[Read the full documentation on the Tilequery API](https://www.mapbox.com/api-documentation/#tilequery)
```
Comment (HeyStenson) [inline src/pages/java/overview/tilequery.md]:
```suggestion
For more information about this API, see the [Mapbox Tilequery API documentation](https://www.mapbox.com/api-documentation/#tilequery).
```

File: src/pages/java/overview/tilequery.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+`}
+/>
+}}
+
+{{
+  <Floater
+    url="https://www.mapbox.com/help/tilequery-api-playground/"
+    title="Tilequery Playground"
+    category="playground"
+    text="Explore how the API to retrieve features from vector tiles."
```
Comment (HeyStenson) [inline src/pages/java/overview/tilequery.md]:
```suggestion
    text="Explore how to use the Tilequery API to retrieve features from vector tiles."
```

File: src/pages/java/overview/tilequery.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  />
+}}
+
+## Query response
+
+You can access the Tilequery API's response inside of the `onResponse` callback. The callback is a Retrofit callback, similar to all of the Mapbox Java SDK's other API calls. The response will include a `List<Feature>` if the query you built has any `Feature`s in it. However, there's no guarantee that the response will have any `Feature` objects in it.
```
Comment (HeyStenson) [inline src/pages/java/overview/tilequery.md]: > The callback is a Retrofit callback, similar to all of the Mapbox Java SDK's other API calls.

I think this sentence could use a little clarification. Would something like "The `onResponse` callback is a Retrofit callback, similar to callbacks received in responses to other API calls to the Mapbox Java SDK." be accurate? And is it possible to expand on how it's similar?

**Pull request merged**